lib/symlink.js

```diff
@@ -31,8 +31,9 @@ function buildLinkPaths(libraries, symlinkPath) {
 function symLink(paths, callback) {
   try {
     paths.forEach(function(p) {
-      if (fs.lstatSync(p[0]).isDirectory()){
-        fse.copySync(p[0], p[1], {overwrite:true, filter: filterFunc });
+      var stat = fs.lstatSync(p[0]); 
+      if (stat.isDirectory() || stat.isSymbolicLink){
```

Comment:
@millzpaugh should `if (stat.isDirectory() || stat.isSymbolicLink){` be `if (stat.isDirectory() || stat.isSymbolicLink()){`?